### PR TITLE
Server summary

### PR DESCRIPTION
**Pull request by Georgehe4**
Moved summarizer to server side.

### CODE REVIEW

**Comment by Georgehe4** [comment]
@devoncarew @lukechurch

File: lib/playground.dart [hunk 1]
```diff
@@ -564,6 +563,19 @@ class Playground implements GistContainer, GistController {
...
+      ..sources['html'] = _context.htmlSource;
+    return dartServices
+        .summarize(input)
+        .timeout(shortServiceCallTimeout)
+        .catchError((e) {
+          _logger.severe(e);
+        });
+  }
+  
+  /*Future<String> _createSummary() {
```
Comment (devoncarew) [inline lib/playground.dart]: perhaps remove this if unused?

**Comment by devoncarew** [comment]
lgtm! Let's definitely _not_ merge this until the corresponding PR on the backend is merged and the change pushed to the live server.

**Comment by lukechurch** [comment]
Sorry, I didn't see this. Lets deal with getting this deployed and tested at after we've landed the export.

**Comment by lukechurch** [comment]
@Georgehe4 This code is in the server now, are you good to bring this back alive?

**Comment by Georgehe4** [comment]
I'll start bringing this in on a new PR soon.

**Pull request closed without merging**